src/Inventory.zig

```diff
@@ -364,7 +371,12 @@ pub const Sync = struct { // MARK: Sync
 
 					const inventoryZon = playerData.getChild(@tagName(source));
 
```

Comment:
I think I may have forgot to remove this entire case here, when I introduced the externally managed method above. Let me investigate this.